integration/src/test/java/org/mule/test/core/context/notification/FlowStackTestCase.java

```diff
@@ -289,7 +289,7 @@ public void subFlowStaticWithScatterGather() throws Exception {
     assertStackElements(stackToAssert,
                         isFlowStackElement("subFlow", "subFlow/processors/0"),
                         isFlowStackElement("subFlowStaticWithScatterGather",
-                                           "subFlowStaticWithScatterGather/processors/0/route/1/processors/0"));
+                                           "subFlowStaticWithScatterGather/processors/0/route/1/processors/0/processors/0"));
```

Comment:
Ignore test instead of update it and put a TODO with an issue. Same in all of this paths. That needs to be fixed this sprint.